lambdas/db-migration/src/migrations/20200901060125_create_files_granules.ts

```diff
@@ -0,0 +1,48 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => knex.schema
+  .createTable('granules', (table) => {
+    table.bigIncrements('cumulusId').comment('Internal Cumulus ID for a granule').primary();
+    table.text('granuleId').comment('Granule ID');
+    table.text('pdrName').comment('PDR associated with the granule');
+    table.integer('collectionCumulusId').references('cumulusId').inTable('collections').notNullable();
+    table.enum('status', ['running', 'completed', 'failed']).comment('Ingest status of the granule');
+    table.text('execution').comment('Step Function Execution link');
+    table.text('cmrLink').comment('Link to granule in the CMR API');
+    table.text('published').comment('Flag that shows if the granule has been published in CMR');
+    table.float('duration').comment('Ingest duration');
+    table.jsonb('error').comment('JSON error object');
+    table.integer('productVolume');
+    table.float('timeToArchive').comment('Number of seconds granule took to archive');
+    table.float('timeToProcess').comment('Number seconds granule took to complete "processing"');
+    table.date('beginningDateTime').comment('Date granule started');
```

Comment:
I'm not sure, but do we want to use `datetime` and not `date` for all these fields? I don't think `date` includes time of day, which we do want. `datetime` is what the `created_at`/`updated_at` fields use by default

Reply:
👍   Correct you are.

Reply:
https://www.postgresql.org/docs/10/datatype-datetime.html

Timestamp seems right.  c9dbb063d